### DISCUSSION TITLE
Build failed

### DISCUSSION THREAD

**Discussion by lyorb** [codemagic-ci-cd/codemagic-docs#1100] [Q&A] [answered]
My build fails at the preparing build machine step, can i have some help ?

![image](https://user-images.githubusercontent.com/15342826/198313415-2b236155-c58b-4aa7-b09d-187493c8afad.png)

**Comment by PieterPicup** [comment]
Exactly the same issue on my side. Kinda glad I'm not the only one

**Comment by himesh-cm** [comment, marked as answer]
Can you please try again?
@PieterPicup @lyorb

**Reply by lyorb** [reply]
It looks good from my side!

Thanks

**Reply by PieterPicup** [reply]
Looks to be sorted. Thanks! Now to fix my build failures 😓
